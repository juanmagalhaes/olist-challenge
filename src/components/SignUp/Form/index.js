// @flow

export { FormContainer as Form } from './Form';
export type {
  State,
  Action,
} from './types';

