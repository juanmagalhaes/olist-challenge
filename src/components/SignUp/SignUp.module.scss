$login-box-width: 515px;
$login-box-height: 740px;
$login-box-background: #FFFFFF;
$login-box-border-color: #F2F2F2;

.box {
  text-align: center;
  width: $login-box-width;
  height: $login-box-height;
  background: $login-box-background;
  border: solid 3px $login-box-border-color;
}

$logo-margin-top: 58px;

.logo {
  margin-top: $logo-margin-top;
}

$heading-size: 22px;
$heading-margin: 20px 0 0;

.headding {
  margin: $heading-margin;
  font-size: $heading-size;
}

