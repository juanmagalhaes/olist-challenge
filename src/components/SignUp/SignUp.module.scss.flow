// @flow
/* This file is automatically generated by css-modules-flow-types */
declare module.exports: {|
  +'box': string;
  +'headding': string;
  +'logo': string;
|};
