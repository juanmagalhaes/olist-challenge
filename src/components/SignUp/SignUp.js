// @flow

import React from 'react';
import { Form } from './Form';
import { Logo } from '../Logo';
import styles from './SignUp.module.scss';

const SignUp = () => (
  <div className={styles.box}>
    <div className={styles.logo}>
      <Logo />
    </div>
    <p className={styles.headding}>
      Crie sua conta
    </p>
    <Form />
  </div>
);

export { SignUp };

