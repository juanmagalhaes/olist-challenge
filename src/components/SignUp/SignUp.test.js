import React from 'react';
import { SignUp } from '~/components/SignUp';

test('SignUp', () => expect(shallow(<SignUp />)).toMatchSnapshot())

