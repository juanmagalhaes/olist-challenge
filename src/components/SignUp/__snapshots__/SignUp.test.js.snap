// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp 1`] = `
<div
  className="box"
>
  <div
    className="logo"
  >
    <Logo />
  </div>
  <p
    className="headding"
  >
    Crie sua conta
  </p>
  <Connect(Form) />
</div>
`;
